src/components/grid/grid.ts

```diff
@@ -392,7 +392,8 @@ export class Grid extends Component<Props, SpreadsheetChildEnv> {
     let prevCol = col;
     let prevRow = row;
 
-    const onMouseMove = (col: HeaderIndex, row: HeaderIndex) => {
+    const onMouseMove = (col: HeaderIndex, row: HeaderIndex, ev: MouseEvent) => {
+      ev.preventDefault();
```

Comment:
actually, maybe add a comment to explain why we prevent default. (especially since there is no test for it)
robodoo r-